README.md

```diff
@@ -1,108 +1,25 @@
-# Ember Widgets by Addepar [![Build Status](https://secure.travis-ci.org/Addepar/ember-widgets.svg?branch=master)](http://travis-ci.org/Addepar/ember-widgets)
```

Comment:
The original file is basically all references to Grunt stuff, I updated it a little but forgot to commit the changes, will push in a sec